JCB_Cinema.Application/Servicies/AppUserService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AutoMapper;
+using JCB_Cinema.Application.DTOs;
+using JCB_Cinema.Application.Interfaces;
+using JCB_Cinema.Application.Requests;
+using JCB_Cinema.Domain.Entities;
+using JCB_Cinema.Domain.Interface;
+using JCB_Cinema.Infrastructure.Data.Interfaces;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace JCB_Cinema.Application.Servicies
+{
+
+    public class AppUserService : ServiceBase, IAppUserService
+    {
+        public AppUserService(IUnitOfWork unitOfWork, IMapper mapper, UserManager<AppUser> userManager, IUserContextService userContextService) : base(unitOfWork, mapper, userManager, userContextService) { }
+
+
+        public async Task<AppUserDTO?> GetAppUserAsync(string id)
```

Comment:
Zrob dedykowany RequestModel
id ma byc nullowalne, dlatego ze dostep do innego niz swojego id bedzie miec uzytkownik Admin, tak to user ma pobierac tylko swoje dane.